main.go

```diff
@@ -104,7 +104,11 @@ func argsParse(argv []string) (finder.Args, error) {
 		args.SRIOVConfigFile = path
 	}
 	args.SysfsRoot = arguments["--sysfs"].(string)
-	args.ContainerRuntime = arguments["--container-runtime"].(string)
+	runtime := arguments["--container-runtime"].(string)
+	if !(runtime == "containerd" || runtime == "cri-o") {
```

Comment:
Its equivalent to : `if runtime != "containerd" && runtime != "cri-o" {`
Can do this too, not really sure which is conventionally better

Reply:
both forms are fine. The one you have in the comments is probably more widespread and thus a bit easier to read, but the other is not wrong. I'd jyst move forward and revisit when we add the (much needed) tests.

Reply:
Okay!